xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -525,3 +525,14 @@
 #macro(glyphicon $iconName)
```

Comment:
As I'm sure you noticed, if you modify a macro in macros.vm, in order to see the change, you need to restart the server. That's why sometimes it's easier to develop / debug using a page with a velocity and html macro, and after apply the changes.

Reply:
Thanks for the tip. :)